src/main/java/mate/controller/carcontroller/CreateCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller.carcontroller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("mate");
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("name");
+        if (!model.isEmpty()) {
+            Car car = new Car();
+            car.setModel(model);
```

Comment:
and where do you create a manufacturer to be passed to car?

Reply:
I didn't create manufacturer to be passed to car, cause task is Create a new controller: Implement creating a car (with no drivers by default). So, I think that I don't need to add it.
Added.